reference-implementation/run-tests.js

```diff
@@ -2,12 +2,14 @@ const glob = require('glob');
 const path = require('path');
 
 import ReadableStream from './lib/readable-stream';
+import { IsReadableStreamDisturbed } from './lib/readable-stream';
 import WritableStream from './lib/writable-stream';
 import ByteLengthQueuingStrategy from './lib/byte-length-queuing-strategy';
 import CountQueuingStrategy from './lib/count-queuing-strategy';
 import TransformStream from './lib/transform-stream';
 
 global.ReadableStream = ReadableStream;
+global.IsReadableStreamDisturbed = IsReadableStreamDisturbed;
```

Comment:
I don't like this testing strategy because the global modifications are meant to be based on https://streams.spec.whatwg.org/#globals. I am not sure what the best strategy for testing it is though. I will try to work on something.